blueprints/authenticator/files/__root__/authenticators/__name__.js

```diff
@@ -0,0 +1,8 @@
+<%= imports %>
+
+export default <%= baseClass %>.extend({
+  <%= properties %>
+  authenticate: function(data) {
+    return Ember.RSVP.resolve(data);
+  }
+});
```

Comment:
This should have the `restore` and `invalidate` methods as well since when inheriting the base authenticator it's mandatory to implement them. Also the default implementation of `authenticate` should just be empty I'd say.